compliance_checker/tests/conftest.py

```diff
@@ -0,0 +1,106 @@
+import os
+import subprocess
+
+from functools import reduce
+from operator import iconcat
+from pathlib import Path
+
+import pytest
+
+from netCDF4 import Dataset
+from pkg_resources import resource_filename
+
+from compliance_checker.cf import CF1_6Check, CF1_7Check, util
+from compliance_checker.suite import CheckSuite
+
+
+def flatten_list(listOfLists):
+    """or list of tups"""
+    return reduce(iconcat, listOfLists, [])
+
+
+def glob_down(pth, suffix, lvls):
+    """globs down up to (lvls: int) levels of subfolders\n
+    suffix in the form ".ipynb"\n
+    pth: Path"""
+    return flatten_list([list(pth.glob(f'*{"/*"*lvl}{suffix}')) for lvl in range(lvls)])
```

Comment:
Minor point, but if this is a list of lists, why not `itertools.chain(*pth.glob...)` here?

Reply:
Ok cool, I didn't realize that does the same thing; I just switched it to chain.